### ISSUE TITLE
arcade.play_sound lags the first time

### ISSUE THREAD

**Issue by alejcas** [pythonarcade/arcade#1367]
## System Info

arcade development (pre 2.7)

### Actual behavior:
The first call to `arcade.play_sound` lags a lot. 
Subsequent calls plays the sound without problems.

### Expected behavior:
There should be no lag playing sounds.

### Steps to reproduce/example code:
Check the platformer tutorial >= "Step 7 coins and sound."

**Comment by inkpoet63** [comment]
I'm having the same issue. I'm thinking about playing a sound as the game opens, to see if that helps

**Comment by einarf** [comment]
It was determined this is due to the first player instance calling `get_audio_driver()` for the first time. Probably in `load_sound` we could do the following:
```py
from pyglet.media import get_audio_driver
get_audio_driver()
```

**Comment by inkpoet63** [comment]
I just tried it and that fixed it for me! Thanks for the help!

**Comment by alejcas** [comment]
Lazy getting the audio driver inside load sound seems the best choice to me

**Comment by einarf** [comment]
Fixed in 10e48a0
